client/src/App.css

```diff
@@ -31,3 +31,25 @@
     transform: rotate(360deg);
   }
 }
+.card {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  width: 300px;
+  height: 200px;
+  background-color: rgb(137, 144, 146);
+  border-radius: 10px;
+  border: 3px solid rgb(187, 191, 194);
+  box-shadow: 0px 5px 15px 1px black;
+}
+.darkMode {
+  background-color: rgb(43, 43, 43);
```

Comment:
To make the text more readable in dark mode, you could give the `color` property a bright value like `white`